### ISSUE TITLE
Form parameters support

### ISSUE THREAD

**Issue by giacgbj** [mulesoft-labs/raml-for-jax-rs#280]
Are form parameters supported using RAML 1.0 from RAML to Java?

For example
```
/foo:
  post:
    body:
      application/x-www-form-urlencoded:
        type:
          properties:
            bar:
              type: string
```

**Comment by jpbelang** [comment]
Not right now.  I'm working on it.  This would lead to @FormParam being added to the method signature.

**Comment by jamalkhattab** [comment]
Will the @FormParm change be also added to the jax-rs-to-raml project?

**Comment by jpbelang** [comment]
I'll try to work on it this weekend.

**Comment by jpbelang** [comment]
I know how to do it.  I'll try to have something simple (basic types) available tomorrow.  I'll see if complex types are possible.

**Comment by jpbelang** [comment]
Could you look at this, please ?  I've only done RAML 1.0 according to this:

https://github.com/raml-org/raml-spec/wiki/Breaking-Changes#defining-form-parameters-has-been-removed-in-favour-for-raml-types

**Comment by jamalkhattab** [comment]
Hey @jpbelang, will your change affect the jax-to-raml project as well?  I downloaded the code from the release/3.0.2 branch, and I used the "jaxrs-to-raml-cli-3.0.2-SNAPSHOT-jar-with-dependencies" jar to generate the Raml file of a JAX-RS API that has parameters using the @FormParam annotation and it didn't work. Should I open an issue under the jax-to-raml project instead to add support for JAX-RS form parameters to Raml?
